src/commands/publish/preview/mod.rs

```diff
@@ -76,7 +77,12 @@ pub fn preview(
             HTTPMethod::Get => get(cookie, &client)?,
             HTTPMethod::Post => post(cookie, &client, body)?,
         };
-        let msg = format!("Your worker responded with: {}", worker_res);
+        let msg = match sites_preview {
+            false => format!("Your Worker responded with: {}", worker_res),
+            true => {
+                "Your Worker is a Workers Site, please preview it in browser window.".to_string()
```

Comment:
nit: I feel like this messaging is a bit awkward but I'm not sure what I'd replace it with. Maybe "This worker can only be previewed in the browser window"

Reply:
I still think it's worth explicitly stating you're using a Workers Site :)